lib/Builder.js

```diff
@@ -178,6 +294,28 @@ class Builder {
             }
         }
     }
+    /**
+     * check if a market name exists in current mcdev config
+     *
+     * @param {string} market market localizations
+     * @returns {boolean} found market or not
+     */
+    static _checkMarket(market) {
```

Comment:
this feels a lot like it should be in util not in builder?

Reply:
yes, but in the utils file, you could avoid getting loadConfigFile, and just pass the properties in. I looked at all references and all have properties object available before using this method. I can imagine it being useful for other items in future elsewhere :/ not about to fall on my sword about it given its only open item - we could reopen in a new point about removing this cyclical dependency item

Reply:
done